src/entity/Ticker.ts

```diff
@@ -22,10 +17,9 @@ export class Ticker extends AbstractBaseEntity {
   name!: string
 
   @Column()
-  @Length(1, 8)
   chainId!: number
 
-  @Column({ nullable: true })
+  @Column({ unique: true })
```

Comment:
There is combined unique constraint for chainId + address, so you don't need unique here (and it allows having same address on different networks)